libs/components/package.json

```
@@ -14,39 +14,67 @@
   "author": "Chainlit",
   "license": "Apache-2.0",
   "version": "0.0.2",
-  "main": "dist/src/index.js",
+  "type": "module",
+  "main": "dist/src/index.cjs",
```

`exports` overrides those properties, but it's contradictory to have both a module type and a main commonjs file.

```suggestion
  "main": "dist/src/index.js",
```